upgrade/upgrade-scripts/upgrade

```diff
@@ -122,8 +122,8 @@ function upgrade_in_place() {
 	#
 	[[ "$DLPX_UPGRADE_DRY_RUN" == "true" ]] && return
 
-	set_upgrade_property "UPGRADE_TYPE" "FULL" ||
-		die "failed to set upgrade property 'UPGRADE_TYPE'"
+	set_upgrade_property "UPGRADE_TYPE" "$UPGRADE_TYPE" ||
+		die "failed to set upgrade property 'UPGRADE_TYPE' to $UPGRADE_TYPE"
```

Comment:
Can we wrap `$UPGRADE_TYPE` with quotes.. it's not necessary, but I find this convention useful since it can denote when the variable's value begins and ends, when the message is printed, which can be useful if the variable contains something unexpected (e.g. when it's empty, or when it has trailing/leading whitespace, etc).